cdap-app-fabric/src/main/java/co/cask/cdap/gateway/handlers/ProgramLifecycleHttpHandler.java

```diff
@@ -555,6 +558,72 @@ public void programSpecification(HttpRequest request, HttpResponder responder,
   }
 
   /**
+   * Update the log level for a running program according to the request body. Currently supported program types are
+   * {@link ProgramType#FLOW}, {@link ProgramType#SERVICE} and {@link ProgramType#WORKER}.
+   * The request body is expected to contain a map of log levels, where key is loggername, value is one of the
+   * valid {@link org.apache.twill.api.logging.LogEntry.Level} or null.
+   */
+  @PUT
+  @Path("/apps/{app-name}/{program-type}/{program-name}/runs/{run-id}/loglevels")
```

Comment:
So this endpoint will change all running versions (Service allows multiple versions running)?